create-cluster/action.yml

```diff
@@ -31,6 +31,9 @@ inputs:
     description: 'Time to wait for the cluster to have a status of `running`'
     required: false
     default: '20'
+  tags:
+    description: 'Tags to assign to the cluster.'
```

Comment:
should this give a hint to the format?

Reply:
I think `description` is single line string. I just added an example in
example-workflows/development-helm.yaml

Reply:
perhaps you should consider a different format if its too hard to document in a single line. how about comma separated key=value?

this action has good descriptions to use as an example

https://github.com/peter-evans/create-pull-request/blob/main/action.yml

Reply:
I guess, multiline strings also work for descriptions.